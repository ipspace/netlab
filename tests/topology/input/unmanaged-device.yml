---
defaults.device: eos
defaults.tests.ignore: [ addressing, defaults, includes, unmanaged ]

nodes:
  r1:
  h1:
    device: linux
  external:
    unmanaged: true

links:
- r1:
  external:
  role: external
- r1:
  h1:
  type: lan
