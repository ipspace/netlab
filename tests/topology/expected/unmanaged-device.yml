input:
- topology/input/unmanaged-device.yml
- package:topology-defaults.yml
links:
- interfaces:
  - ifindex: 1
    ifname: Ethernet1
    ipv4: 10.1.0.2/30
    node: r1
  - ifindex: 1
    ifname: Ethernet1
    ipv4: 10.1.0.1/30
    node: external
  linkindex: 1
  node_count: 2
  prefix:
    ipv4: 10.1.0.0/30
  role: external
  type: p2p
- bridge: input_2
  gateway:
    ipv4: 172.16.0.1/24
  interfaces:
  - ifindex: 2
    ifname: Ethernet2
    ipv4: 172.16.0.1/24
    node: r1
  - ifindex: 1
    ifname: eth1
    ipv4: 172.16.0.2/24
    node: h1
  linkindex: 2
  node_count: 2
  prefix:
    ipv4: 172.16.0.0/24
  role: stub
  type: lan
name: input
nodes:
  h1:
    af:
      ipv4: true
    box: generic/ubuntu2004
    device: linux
    id: 2
    interfaces:
    - bridge: input_2
      gateway:
        ipv4: 172.16.0.1/24
      ifindex: 1
      ifname: eth1
      ipv4: 172.16.0.2/24
      linkindex: 2
      name: h1 -> r1
      neighbors:
      - ifname: Ethernet2
        ipv4: 172.16.0.1/24
        node: r1
      role: stub
      type: lan
    mgmt:
      ifname: eth0
      ipv4: 192.168.121.102
      mac: 08-4F-A9-00-00-02
    name: h1
    role: host
  r1:
    af:
      ipv4: true
    box: arista/veos
    device: eos
    id: 1
    interfaces:
    - ifindex: 1
      ifname: Ethernet1
      ipv4: 10.1.0.2/30
      linkindex: 1
      name: r1 -> external
      neighbors:
      - ifname: Ethernet1
        ipv4: 10.1.0.1/30
        node: external
      role: external
      type: p2p
    - bridge: input_2
      ifindex: 2
      ifname: Ethernet2
      ipv4: 172.16.0.1/24
      linkindex: 2
      name: r1 -> h1
      neighbors:
      - ifname: eth1
        ipv4: 172.16.0.2/24
        node: h1
      role: stub
      type: lan
    loopback:
      ipv4: 10.0.0.1/32
    mgmt:
      ifname: Management1
      ipv4: 192.168.121.101
      mac: 08-4F-A9-00-00-01
    name: r1
provider: libvirt
